ckanext-notify/README.rst

```diff
@@ -0,0 +1,166 @@
+
+=============
+ckanext-notify
+=============
+
+.. Put a description of your extension here:
+   What does it do? What features does it have?
+   Consider including some screenshots or embedding a video!
+
+
+------------
+Requirements
+------------
+
+For example, you might want to mention here which versions of CKAN this
+extension works with.
+
+
+------------
+Installation
+------------
+
+.. Add any additional install steps to the list below.
+   For example installing any non-Python dependencies or adding any required
+   config settings.
+
+To install ckanext-notify:
+
+1. Activate your CKAN virtual environment, for example::
+
+     . /usr/lib/ckan/default/bin/activate
+
+2. Install the ckanext-notify Python package into your virtual environment::
+
+     pip install ckanext-notify
+
+3. Add ``notify`` to the ``ckan.plugins`` setting in your CKAN
+   config file (by default the config file is located at
+   ``/etc/ckan/default/production.ini``).
+
+4. Restart CKAN. For example if you've deployed CKAN with Apache on Ubuntu::
+
+     sudo service apache2 reload
+
+
+---------------
+Config Settings
+---------------
+
+Document any optional config settings here. For example::
+
+    # The minimum number of hours to wait before re-checking a resource
+    # (optional, default: 24).
+    ckanext.notify.some_setting = some_default_value
+
+
+------------------------
+Development Installation
+------------------------
+
+To install ckanext-notify for development, activate your CKAN virtualenv and
+do::
+
+    git clone https://github.com/"WNjihia"/ckanext-notify.git
+    cd ckanext-notify
+    python setup.py develop
+    pip install -r dev-requirements.txt
+
+
+-----------------
+Running the Tests
+-----------------
+
+To run the tests, do::
+
+    nosetests --nologcapture --with-pylons=test.ini
+
+To run the tests and produce a coverage report, first make sure you have
+coverage installed in your virtualenv (``pip install coverage``) then run::
+
+    nosetests --nologcapture --with-pylons=test.ini --with-coverage --cover-package=ckanext.notify --cover-inclusive --cover-erase --cover-tests
+
+
+---------------------------------
+Registering ckanext-notify on PyPI
+---------------------------------
+
+ckanext-notify should be availabe on PyPI as
+https://pypi.python.org/pypi/ckanext-notify. If that link doesn't work, then
+you can register the project on PyPI for the first time by following these
+steps:
+
+1. Create a source distribution of the project::
+
+     python setup.py sdist
+
+2. Register the project::
+
+     python setup.py register
+
+3. Upload the source distribution to PyPI::
+
+     python setup.py sdist upload
+
+4. Tag the first release of the project on GitHub with the version number from
+   the ``setup.py`` file. For example if the version number in ``setup.py`` is
+   0.0.1 then do::
+
+       git tag 0.0.1
+       git push --tags
+
+
+----------------------------------------
+Releasing a New Version of ckanext-notify
```

Comment:
I feel like releasing a new version of notify is your work as the creator of notify and not the work of your users, I would advise that this is not included on the Readme, the readme is always to help people set up the extension and be able to configure it for its intended purpose and not to release new versions -if contribution guide would be added, remember you would be the maintainer taking pull requests then building this new versions by yourself.